MicrosoftTeams.schema.json

```diff
@@ -155,6 +155,22 @@
                                 "groupchat"
                             ]
                         }
+                    },
+                    "sharePointPreviewImage": {
+                        "type": "string",
+                        "description": "A relative file path to a tab preview image for use in SharePoint. Size 1024x768.",
+                        "maxLength": 2048
+                    },
+                    "supportedSharePointHosts": {
+                        "type": "array",
+                        "description": "Defines how your tab will be made available in SharePoint.",
+                        "maxItems": 2,
```

Comment:
I would add uniqueItems: true also

Reply:
OK, added. There are a number of other arrays in there that should probably have this property added as well but that's out of scope for this PR. We should consider adding those in a separate change.